week-2/E-conditionals/exercise.js

```diff
@@ -1,24 +1,16 @@
 /*
   Conditionals
   ---------------------------------
-  Add an if statement to only show the name if the role is student
+  Add an if statement to check Daniel's role in a CYF class.
```

Comment:
Changed, as students (and myself actually) found it a little confusing - which role to check, do I check both, should I try to print both, etc.